java-frontend/src/main/java/org/sonar/java/ast/JavaAstScanner.java

```diff
@@ -64,14 +66,18 @@ public void scan(Iterable<? extends InputFile> inputFiles) {
     boolean successfullyCompleted = false;
     boolean cancelled = false;
     try {
+      ExecutionTimeReport executionTimeReport = new ExecutionTimeReport(Clock.systemUTC());
       for (InputFile inputFile : inputFiles) {
         if (analysisCancelled()) {
           cancelled = true;
           break;
         }
+        executionTimeReport.start(inputFile.uri().getPath());
```

Comment:
I would use the inputFile.toString(), it displays a readable understandable by end users

Reply:
ok, I will try

Reply:
good try, paths are more readable:
```
[INFO] Slowest analyzed files: src/main/java/org/sonar/server/issue/IssueQueryFactory.java (2223ms), src/main/java/org/sonar/ce/CeModule.java (1507ms), src/main/java/org/sonar/server/rule/RegisterRules.java (1298ms)
```